aldryn_config.py

```diff
@@ -15,21 +16,18 @@ def _show_toolbar(self, request):
         return True
 
     def to_settings(self, data, settings):
-        enable_debug_toolbar = data['enable_debug_toolbar']
 
-        from aldryn_addons.utils import djsenv
-        env = partial(djsenv, settings=settings)
-        env_stage = '' if env('STAGE') is None else str(env('STAGE'))
-        development_mode = True if env_stage in ['test', 'local'] else False
+        if settings["DEBUG"] and data['enable_debug_toolbar']:
+
+            settings["INSTALLED_APPS"].extend(["debug_toolbar"])
+
+            settings["DEBUG_TOOLBAR_CONFIG"] = {"SHOW_TOOLBAR_CALLBACK": self._show_toolbar}
+
+            settings["MIDDLEWARE_CLASSES"].insert(
+                settings["MIDDLEWARE_CLASSES"].index("django.middleware.gzip.GZipMiddleware") + 1,
```

Comment:
This will throw an ValueError if ``django.middleware.gzip.GZipMiddleware`` is not in the list.